paddle/framework/net_design.md

```diff
@@ -0,0 +1,251 @@
+# Network Design
+
+`Network` is the container and controller of a set of operators,
```

Comment:
Will the framework have an independent `Executor`? 
or merge the `Executor` into `Network'?  @wangkuiyi